pkg/controller/node/nodecontroller.go

```diff
@@ -115,7 +115,7 @@ type NodeController struct {
 	allocatorType     CIDRAllocatorType
 
 	cloud        cloudprovider.Interface
-	clusterCIDR  *net.IPNet
+	clusterCIDR  []*net.IPNet
```

Comment:
with the older version , it's just a slice with one element. It seemed simpler this way that to have a single element along with the new slice